src/create-glamorous.js

```diff
@@ -149,6 +149,13 @@ function createGlamorous(splitProps) {
         },
       })
 
+      function withComponent(newComp, options) {
+        return glamorous(newComp, {
+          forwardProps: GlamorousComponent.forwardProps,
+          ...options,
```

Comment:
Oh you know.. if you were to pass it as a rest parameter, i think that's when it might:

```js
var obj = undefined
method(...obj)
```